src/test/java/org/folio/rest/impl/NotesTest.java

```diff
@@ -158,8 +158,7 @@ public static void setUpBeforeClass(TestContext context) {
           .body(tenants)
           .post("/_/tenant")
           .then()
-          .log().ifValidationFails()
-          .statusCode(201);
```

Comment:
Check for status code 201 in test setUp fails when multiple tests run at the same time, 
because embedded postgres always uses port 6000, so tenant already exists in some cases, 
it should be fine to run tests locally against already running postgres db, so I think it's better to remove this check